package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "ember-blog",
+  "version": "1.0.0",
+  "description": "This repository contains the blog content for the Ember Blog.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ember-learn/ember-blog.git"
+  },
+  "author": "",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ember-learn/ember-blog/issues"
+  },
+  "homepage": "https://github.com/ember-learn/ember-blog#readme",
+  "scripts": {
+    "lint:alex": "alex source",
+    "lint:md": "markdownlint source"
```

Comment:
I believe `validate` shell script only lints files that were modified in a branch.

The `lint:alex` and `lint:md` scripts, on the hand, lints all files in the `source` directory. Currently, if we run these scripts, we will see many errors. Because of this, I didn't update the workflow to run `yarn lint` (assuming `npm-run-all` is installed also).